### ISSUE TITLE
Pitch shifting and time stretching in a single transform

### ISSUE THREAD

**Issue by iver56** [asteroid-team/torch-audiomentations#101]
A transform that does both in one go, so it's faster (and probably sounds better) than doing pitch shifting and time stretching separately.

**Comment by iver56** [comment]
When I resample, I can get two outcomes:

* The pitch goes up and the tempo goes up
* The pitch goes down and the tempo goes down

Here's a rough illustration with a pink line that shows all the tempo + pitch combinations that we can get by resampling:

![bilde](https://user-images.githubusercontent.com/1470603/137688566-fa9fa192-31d5-4592-8ce4-614b3ce06c60.png)

On the other hand, doing pitch shifting and time stretching allows one to get any combination of tempo and pitch (here illustrated by the blue rectangle). For example, I can pitch down the audio while speeding up the tempo slightly

**Comment by WhiteTeaDragon** [comment]
Is time stretching available in torch-audiomentations? I see only PitchShift augmentation which does not change the tempo

**Comment by iver56** [comment]
No, not at the moment. I recommend using the TimeStretch in audiomentations for the time being.

There is an experimental time stretch PR in torch-audiomentations, but it is nowhere near complete - it requires some changes in the "backbone" of torch-audiomentations
